client/src/Pages/Register/index.js

```diff
@@ -0,0 +1,173 @@
+          }}
+        >
+          <Form.Item
+            label="Name"
+            name="username"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your Name!',
+              },
+              {
+                pattern: /^[a-z]([-']?[a-z]+)*( [a-z]([-']?[a-z]+)*)+$/,
+                message: 'Must be two parts.',
+              },
+            ]}
+          >
+            <Input placeholder="Enter your name..." />
+          </Form.Item>
+          <Form.Item
+            label="Email"
+            name="email"
+            rules={[
+              {
+                type: 'email',
+                message: 'Please enter a valid email!',
+              },
+              {
+                required: true,
+                message: 'Please input your Email!',
+              },
+            ]}
+          >
+            <Input placeholder="Enter a valid email..." />
+          </Form.Item>
+          <Form.Item
+            label="Mobile"
+            name="mobile"
+            rules={[
+              {
+                required: true,
+                message: 'Please input mobile number!',
+              },
+            ]}
+          >
+            <Input placeholder="Enter your mobile number..." />
+          </Form.Item>
+          <Form.Item
+            label="Password"
+            name="password"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your password!',
+              },
+            ]}
+          >
+            <Input type="password" placeholder="Enter password..." />
+          </Form.Item>
+          <Form.Item
+            label="Confirm Password"
+            name="confirmPassword"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your the same password!',
+              },
+            ]}
+          >
+            <Input type="password" placeholder="Confirm the password..." />
+          </Form.Item>
+          <Form.Item
+            className="select-input"
+            name="location"
+            label="Location"
+            rules={[
+              {
+                required: true,
+              },
+            ]}
+          >
+            <Select placeholder="Location" allowClear size="small">
+              <Option value="gaza">Gaza</Option>
+              <Option value="khan Yunis">khan Yunis</Option>
+              <Option value="rafah">Rafah</Option>
+            </Select>
+          </Form.Item>
+          <Form.Item className="select-input" name="role" label="Role">
+            <Radio.Group>
+              <Radio className="radio-label" value="customer">
+                Customer
+              </Radio>
+              <Radio className="radio-label" value="provider">
+                Craftsman
+              </Radio>
+            </Radio.Group>
+          </Form.Item>
+          {error && {
+            error,
+          }}
```

Comment:
I think you don't test your code, this code will throw an error
* use [alert](https://ant.design/components/alert/#header) component to view the error